packages/sst/src/constructs/NextjsSite.ts

```diff
@@ -88,17 +89,18 @@ export class NextjsSite extends SsrSite {
       handler = "index.handler";
     }
 
-    return new lambda.Function(this, `ServerFunction`, {
+    return new Function(this, `ServerFunction`, {
       description: "Server handler for Next.js",
       handler,
       currentVersionOptions: {
         removalPolicy: RemovalPolicy.DESTROY,
       },
-      logRetention: logs.RetentionDays.THREE_DAYS,
+      logRetention: 'three_days',
       code: lambda.Code.fromAsset(bundlePath),
-      runtime: lambda.Runtime.NODEJS_18_X,
+      architecture: defaults?.function?.architecture,
```

Comment:
fallback value?

Reply:
It should default to `undefined`, which should use `x86` by default.

Reply:
@khuezy is nodejs18 runtime required?

Reply:
We're defaulting to the last version of node 18, (to use fetch and newer APIs)